### DISCUSSION TITLE
Figma integration

### DISCUSSION THREAD

**Discussion by BastiaensFer** [vmware-clarity/ng-clarity#1537] [Q&A] [answered]
I cannot seem to find the Clarity Figma libraries. When following the [link ](https://www.figma.com/@vmware) in the documentation I land on a 404.  Are these removed?

**Comment by kevinbuhmann** [comment, marked as answer]
Broadcom disabled all password-less public Figma links. I have raised the need to update our public website, so this is a known issue.

**Reply by BastiaensFer** [reply]
Will the Clarity Figma library ever be publicly available again?

**Reply by kevinbuhmann** [reply]
I don't know.

**Reply by hxs7** [reply]
Is there an archived open source version of the Figma libraries?

**Reply by kevinbuhmann** [reply]
Not at this time.
